doc/src/base/numbers.md

```diff
@@ -113,18 +111,21 @@ Base.@int128_str
 Base.@uint128_str
 ```
 
-## BigFloats
+## BigFloats and BigInts
 
 The [`BigFloat`](@ref) type implements arbitrary-precision floating-point arithmetic using
-the [GNU MPFR library](http://www.mpfr.org/).
+the [GNU MPFR library](http://www.mpfr.org/) and [`BigInt`](@ref) implements
```

Comment:
I think it might be slightly easier to read if these were two sentences; alternatively I'd add a comma or semicolon before the "and", to emphasize the separation of the clauses.